index.html

```diff
@@ -3476,6 +3606,49 @@ <h2>
           </li>
         </ol>
       </section>
+      <section>
+        <h2>
+          Payment method changed algorithm
+        </h2>
+        <p>
+          The user agent MUST run the <dfn>payment method changed
+          algorithm</dfn> runs when the user changes <a>payment method</a>. The
+          algorithm takes <var>methodDetails</var>, which is either
+          <a data-cite="!WEBIDL#idl-dictionary">dictionary</a> or an
+          <a data-cite="!WEBIDL#idl-object">object</a> or null, and a
+          <var>methodName</var>, which is a DOMString that represents the
+          <a>payment method identifier</a> of the <a>payment handler</a> the
+          user is interacting with:
```

Comment:
How does the user agent determine _methodName_ and _methodDetails_?

Reply:
Will rephrase this, as it's actually the Payment Handler that needs to invoke this. From discussions over in Basic Card, we are going to make this optional... so Basic Card won't fire this, but Apply Pay will.